BuildTransportPackage.ps1

```diff
@@ -0,0 +1,170 @@
+Param(
+    [string]$Platform,
+    [string]$Configuration,
+    [string]$LocalPackagesPath = $null,
+    [string]$UpdateVersionDetailsPath = $null
+)
+
+$MRTSourcesDirectory = "dev\MRTCore"
+$MRTBinariesDirectory = "BuildOutput"
+$nuSpecsPath = "build\NuSpecs"
+$fullNugetPath = "build\fullnuget"
+$buildOverridePath = "build\override"
+$windowsAppSdkBinariesPath = "build\windowsappsdk_binaries"
+
+$builddate_yymm = Get-Date -Format "yyMM"
+$builddate_dd = Get-Date -Format "dd"
+$revision = Get-Date -Format "HHMM"
+
+$VCToolsInstallDir = . "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe" -Latest -requires Microsoft.Component.MSBuild -property InstallationPath
+write-host "VCToolsInstallDir: $VCToolsInstallDir"
+
+# do all the directory making before any building.
+if(-not (test-path ".nuget"))
+{
+	 new-item -path ".nuget" -itemtype directory
```

Comment:
There seems to be a tab followed by a space here

Reply:
Fixed.  Also changed all tabs to spaces.